nhooks.lisp

```diff
@@ -38,14 +38,18 @@ If the handler is meant to be a setter, VALUE can be used to describe what FN is
 going to set to PLACE.
 In particular, PLACE and VALUE can be used to compare handlers.
 This can be left empty if the handler is not a setter."))
+  (:metaclass closer-mop:funcallable-standard-class)
   (:documentation "Handlers are wrappers around functions used in typed hooks.
 They serve two purposes as opposed to regular functions:
 
 - They can embed a NAME so that anonymous functions can be conveniently used in hooks.
 - If the handler is meant to be a setter, the PLACE and VALUE slots can be used
   to identify and compare setters.
 
-With this extra information, it's possible to compare handlers and, in particular, avoid duplicates in hooks."))
+With this extra information, it's possible to compare handlers and, in particular, avoid duplicates in hooks.
+
+Handlers are `funcall'able. If you subclass those, don't forget to add
+a `closer-mop:funcallable-standard-class' to retain this property."))
```

Comment:
No need to mention this, subclasses do not have a choice, they _must_ use a compatible metaclass and the default one won't do!  Try it, the Lisp will throw an error.

Reply:
The errors I've got were quite cryptic, so it wouldn't hurt mentioning the exact thing.